README.md

```diff
@@ -159,6 +159,40 @@ Stacktrace:
 [...]
 ```
 
+### Breakpoint at specific location
+It is sometimes useful to programmatically decide when to break and drop to the debugger to inspect your surroundings; this is for instance done in Matlab/Octave with the `keyboard` command. You can use the `@bp` macro in JuliaInterpreter to do this:
```

Comment:
`@bp` is reexported so no need to mention (or import) JuliaInterpreter

Reply:
Also, note that you can have conditional breakpoints. So `bp add f:2 x < 0` or something to break on line 2 if `x < 0`. But perhaps you already know that.

Reply:
Yeah I know, but this `keyboard` thing is really convenient for more complicated codepaths; plus I often find it easier to do things in the source code than in the REPL. I'll mention that this is an alternative.

Reply:
I changed to make clear that the main motivation is to put the breakpoint in the source, not really conditional breakpoints. Also added a ref to R which has a similar thing.